### PR TITLE
feat(sonarr): add SYFY streaming network

### PR DESCRIPTION
**Pull request by mynameisbogdan**
# Pull Request

## Purpose

- Add SYFY Streaming Service Custom Format

## Approach

- Create SYFY Streaming Service Custom Format
- Add to guide pages
- Add to profile JSON files

## Requirements

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Review by TRaSH- — CHANGES_REQUESTED** [review]
Thnx for your PR

Things that needs to be done before merging:
- [ ] Add to `includes/cf/sonarr-streaming-services.md` so it shows up in the guide [How to set up Quality Profiles](https://trash-guides.info/Sonarr/sonarr-setup-quality-profiles/) 
- [ ] Add to `includes/cf/sonarr-uhd-streaming-services.md` so it shows up in the guide [How to set up Quality Profiles](https://trash-guides.info/Sonarr/sonarr-setup-quality-profiles/)
- [ ] Add to `docs/json/sonarr/quality-profiles/web-1080p.json` for 3rd party sync apps
- [ ] Add to `docs/json/sonarr/quality-profiles/web-2160p.json` for 3rd party sync apps

File: docs/Sonarr/sonarr-collection-of-custom-formats.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ We've made 3 guides related to this.
 ---
 
 | General Streaming Services | French Streaming Services | Asian Streaming Services | Dutch Streaming Services |
-| -------------------------- | ------------------------- | ------------------------ | ------------------------ |
+|----------------------------|---------------------------|--------------------------|--------------------------|
```
Comment (bakerboy448) [inline docs/Sonarr/sonarr-collection-of-custom-formats.md:63]: ?

File: includes/cf/sonarr-streaming-services.md [hunk 2]
```diff
@@ -17,3 +17,4 @@
     | [{{ sonarr['cf']['pcok']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#pcok)     |  {{ sonarr['cf']['pcok']['trash_scores']['default'] }}  | {{ sonarr['cf']['pcok']['trash_id'] }}   |
     | [{{ sonarr['cf']['sho']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#sho)       |  {{ sonarr['cf']['sho']['trash_scores']['default'] }}   | {{ sonarr['cf']['sho']['trash_id'] }}    |
     | [{{ sonarr['cf']['stan']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#stan)     |  {{ sonarr['cf']['stan']['trash_scores']['default'] }}  | {{ sonarr['cf']['stan']['trash_id'] }}   |
+    | [{{ sonarr['cf']['syfy']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#syfy)     |  0  | {{ sonarr['cf']['syfy']['trash_id'] }}   |
```
Comment (bakerboy448) [inline includes/cf/sonarr-streaming-services.md:20]: Thought we were doing dynamic scores for all for maintainability

File: docs/Sonarr/sonarr-collection-of-custom-formats.md (hunk 1 above)
Comment (yammes08) [inline docs/Sonarr/sonarr-collection-of-custom-formats.md:63]:
```suggestion
| -------------------------- | ------------------------- | ------------------------ | ------------------------ |
```

Comment (yammes08) [reply docs/Sonarr/sonarr-collection-of-custom-formats.md:63]: Added a reversion.

File: docs/json/sonarr/cf/syfy.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+  "trash_id": "9623c5c9cac8e939c1b9aedd32f640bf",
```
Comment (yammes08) [inline docs/json/sonarr/cf/syfy.json:2]:
```suggestion
  "trash_id": "9623c5c9cac8e939c1b9aedd32f640bf",
  "trash_scores": {
    "default": 50
  },
```

Comment (yammes08) [reply docs/json/sonarr/cf/syfy.json:2]: Streaming Services for Sonarr get a default score of 50

File: includes/cf/sonarr-streaming-services.md (hunk 2 above)
Comment (yammes08) [reply includes/cf/sonarr-streaming-services.md:20]: We are indeed

Comment (yammes08) [inline includes/cf/sonarr-streaming-services.md:20]:
```suggestion
    | [{{ sonarr['cf']['syfy']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#syfy)     |  {{ sonarr['cf']['syfy']['trash_scores']['default'] }}  | {{ sonarr['cf']['syfy']['trash_id'] }}   |
```

File: includes/cf/sonarr-uhd-streaming-services.md [hunk 4]
```diff
@@ -17,6 +17,7 @@
     | [{{ sonarr['cf']['pcok']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#pcok)                                         |        {{ sonarr['cf']['pcok']['trash_scores']['default'] }}         | {{ sonarr['cf']['pcok']['trash_id'] }}                |
     | [{{ sonarr['cf']['sho']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#sho)                                           |         {{ sonarr['cf']['sho']['trash_scores']['default'] }}         | {{ sonarr['cf']['sho']['trash_id'] }}                 |
     | [{{ sonarr['cf']['stan']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#stan)                                         |        {{ sonarr['cf']['stan']['trash_scores']['default'] }}         | {{ sonarr['cf']['stan']['trash_id'] }}                |
+    | [{{ sonarr['cf']['syfy']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#syfy)                                         |        0         | {{ sonarr['cf']['syfy']['trash_id'] }}                |
```
Comment (yammes08) [inline includes/cf/sonarr-uhd-streaming-services.md:20]:
```suggestion
    | [{{ sonarr['cf']['syfy']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#syfy)                                         |  {{ sonarr['cf']['syfy']['trash_scores']['default'] }}         | {{ sonarr['cf']['syfy']['trash_id'] }}                |
```

**Review by yammes08 — CHANGES_REQUESTED** [review]
Sonarr streaming service all get a base score of 50 - see amendments.

**Review by yammes08 — APPROVED** [review]

**Pull request merged**